pkg/workflows/wasm/host/module.go

```diff
@@ -69,24 +70,26 @@ func (r *store) delete(id string) {
 }
 
 var (
-	defaultTickInterval = 100 * time.Millisecond
-	defaultTimeout      = 2 * time.Second
-	defaultMaxMemoryMBs = 256
-	DefaultInitialFuel  = uint64(100_000_000)
+	defaultTickInterval     = 100 * time.Millisecond
+	defaultTimeout          = 2 * time.Second
+	defaultMaxMemoryMBs     = 256
+	DefaultInitialFuel      = uint64(100_000_000)
+	defaultMaxFetchRequests = int64(5)
```

Comment:
@agparadiso This can just be an int I think: it's only used by our code, not by wasmtime-go (which would require an int64, and is why we use that for MaxMemoryMBs for example).